andela/wger-sparta
# #151593397 Update muscle list

### What does this PR do?
Updates list of available muscles, as well as the exercises.

### Description of Task to be Completed.
Add the following new muscles to the muscle fixtures:
lateral deltoid - 17
Obliquus externus abdominis - 18
Posterious Deltoid - 19
Forearms - 20
Also, the added muscles should be matched to their relevant exercises.
Furthermore, the muscles have been drawn and mapped to the body on the sculpture on the left side of the screen.

### How should this be manually tested?
Since this task did not do any modifications to the main code logic. You just need to run the application and view the *muscle overview* tab to see the new fixtures. Note that the fixtures have even their images done.

### Any background context or concerns that need to be provided?
The *Oblisquus externus abdominis* has not been added since it already existed.

### What are the relevant pivotal tracker stories?
#151593397

### Screenshots
###### Image 1: The Forearms
<img width="1280" alt="screen shot 2017-10-10 at 12 29 25 pm" src="https://user-images.githubusercontent.com/17453166/31379408-d0a0c058-adb6-11e7-97dd-e36974cb1df6.png">

###### Image 2: The Posterious Deltoid
<img width="1280" alt="screen shot 2017-10-10 at 12 29 41 pm" src="https://user-images.githubusercontent.com/17453166/31379416-d93ae464-adb6-11e7-958b-38a2b7f8ec3c.png">

###### Image 3: The Lateral Deltoid
<img width="1280" alt="screen shot 2017-10-10 at 12 34 35 pm" src="https://user-images.githubusercontent.com/17453166/31379644-8b0cd738-adb7-11e7-9b46-c4f59dcbfdf3.png">
